src/extensions/nativeExtensions/image/ImageWindow.vue

```diff
@@ -172,6 +160,7 @@ export default class ImageWindow extends mixins(I18nMixin) {
   inputPreviewSources: ImageSource[] = []
 
   get imageTabs () {
+    console.log(this);
```

Comment:
This can be removed.

Reply:
Agreed. Had it there for debugging purposes.